gomint-server/src/main/java/io/gomint/server/entity/monster/EntityTropicalFish.java

```diff
@@ -0,0 +1,41 @@
+package io.gomint.server.entity.monster;
+
+import io.gomint.server.entity.Attribute;
+import io.gomint.server.entity.EntityLiving;
+import io.gomint.server.entity.EntityType;
+import io.gomint.server.registry.RegisterInfo;
+import io.gomint.server.world.WorldAdapter;
+
+@RegisterInfo( id = 111 )
+public class EntityTropicalFish extends EntityLiving implements io.gomint.entity.monster.EntityTropicalFish {
+
+    /**
+     * Constructs a new EntityLiving
+     *
+     * @param world The world in which this entity is in
+     */
+    public EntityTropicalFish( WorldAdapter world ) {
+        super( EntityType.PUFFERFISH, world );
+        this.initEntity();
+    }
+
+    /**
+     * Create new entity salmon for API
```

Comment:
Should be entity tropical fish

Reply:
Yes xD I am sorry, but here: https://github.com/GoMint/GoMint/commit/277df223c83114a5ac954bb3b5354c9071263d14#diff-0d78afa2ec163f820f1c3a98dd9a73e4 is Standing "entity chicken", i think thats not so important :)